packages/upload/test/keyboard-navigation.common.js

```diff
@@ -31,61 +31,108 @@ describe('keyboard navigation', () => {
     uploadElement.files = [FAKE_FILE];
 
     await nextRender();
-
-    fileElement = uploadElement.querySelector('vaadin-upload-file');
+    uploadButton = document.querySelector('vaadin-button[slot=add-button]');
+    fileElements = uploadElement.querySelector('vaadin-upload-file');
   });
 
   afterEach(() => {
     button.focus();
   });
 
   it('should focus on the upload button', async () => {
-    const uploadButton = uploadElement.shadowRoot.querySelector('[part=upload-button]');
-
     await repeatTab(1);
 
-    expect(uploadElement.shadowRoot.activeElement).to.equal(uploadButton);
+    expect(document.activeElement).to.not.equal(null);
+    expect(document.activeElement).to.equal(uploadButton);
   });
 
   it('should focus on the file', async () => {
     await repeatTab(2);
 
-    expect(document.activeElement).to.equal(fileElement);
+    expect(document.activeElement).to.equal(fileElements);
+    expect(document.activeElement).to.not.equal(null);
```

Comment:
There is no need to check for `null`. Also, if you keep `fileElement` variable name as suggested above, you can leave this test unchanged.